src/ui/profile_preferences.ui

```diff
@@ -1981,16 +1981,6 @@ you can use it but there could be issues with aligning columns of text</string>
           <string>Map view</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_groupBox_mapOptions">
-          <item row="0" column="0">
```

Comment:
if you can make sense of it, please restore the <item row"X" column="Y"> (sort by ascending row and then by ascending column in each row) order of elements in this file - unfortunately the Qt Designer plugin (to the Qt IDE) / standalone utility does not take care to maintain the ordering of `QGridLayout`s which balls-up the Git differences.

If you need a hand with this (I use the `plain text edit` editor to do it myself) let me know and i will help...